x11-misc/shutter/shutter-0.95_pre1.ebuild

```diff
@@ -3,22 +3,23 @@
 
 EAPI=7
 
-inherit desktop eutils xdg-utils
+inherit desktop eutils vcs-snapshot xdg-utils
```

Comment:
Per convention we use `COMMIT=af237e7f415a5fa20402703490c6190dc1af9b20` here and set `S="${WORKDIR}/${PN}-${COMMIT}"` below, then don't need to inherit `vcs-snapshot` as github gives us a predictable directory name.

Reply:
Okay... It's shorter this way though, and is as understandable :(